src/core/lib/layer.js

```diff
@@ -133,13 +132,6 @@ export default class Layer {
     return this._getNeedsRedraw(clearRedrawFlags);
   }
 
-  // Checks if layer attributes needs updating
-  needsUpdate() {
```

Comment:
Ok. The half-baked idea here was that multiple classes might expose the same set of methods, `needsRedraw` and `needsUpdate`